src/qforte/system/molecular_info.py

```diff
@@ -120,19 +134,10 @@ def set_cisd_energy(self, cisd_energy):
     def set_ccsd_energy(self, ccsd_energy):
         self._ccsd_energy = ccsd_energy
 
-    def set_fci_energy(self, fci_energy):
-        self._fci_energy = fci_energy
-
 
     def get_ccsd_amps(self):
         return self._ccsd_singles, self._ccsd_doubles
 
-    def get_hamiltonian(self):
-        return self._hamiltonian
-
-    def get_sq_hamiltonian(self):
-        return self._sq_hamiltonian
-
     def get_sq_of_ham(self):
         return self._sq_of_ham
```

Comment:
Should this function be included in the System class and removed from the Molecule class, as was the case with set_sq_of_ham?

Reply:
Good call. Will fix that, then merge this in.